File: src/main/java/org/radarbase/appserver/service/transmitter/NotificationTransmitter.java
package org.radarbase.appserver.service.transmitter;

import org.radarbase.appserver.entity.Notification;
import org.radarbase.appserver.exception.MessageTransmitException;

public interface NotificationTransmitter {
    // Can Sneakily throw MessageTransmitException
    void send(Notification notification);
}

File: appserver-jersey/src/main/kotlin/org/radarbase/appserver/jersey/service/transmitter/NotificationTransmitter.kt
package org.radarbase.appserver.jersey.service.transmitter

import org.radarbase.appserver.jersey.entity.Notification
import org.radarbase.appserver.jersey.exception.MessageTransmitException

interface NotificationTransmitter {
    @Throws(MessageTransmitException::class)
    fun send(notification: Notification)
}
